beacon-chain/rpc/prysm/v1alpha1/beacon/blocks.go

```diff
@@ -134,6 +134,12 @@ func convertToBlockContainer(blk block.SignedBeaconBlock, root [32]byte, isCanon
 			return nil, err
 		}
 		ctr.Block = &ethpb.BeaconBlockContainer_AltairBlock{AltairBlock: rBlk}
+	case version.Bellatrix:
+		rBlk, err := blk.PbBellatrixBlock()
+		if err != nil {
+			return nil, err
+		}
+		ctr.Block = &ethpb.BeaconBlockContainer_BellatrixBlock{BellatrixBlock: rBlk}
```

Comment:
I understand why these API additions are needed for e2e, but they should have standalone unit tests. I'm ok with issue now as long as they can get tracked properly

Reply:
I think when we merge this into develop, we can have them added in with proper unit tests.

Reply:
I think we should target this to `develop`. `kiln` only has 60 lines diff, that's go.mod and go.sum. Everything we are working on will target `develop` from no. Merging this to `kiln` doesn't serve much purpose IMO. I understand we need to go.mod changes, is there anything we can do?

On another topic, everything is red with this PR but everything is green in Kiln branch. Can we fix it?
<img width="620" alt="Screen Shot 2022-03-20 at 8 58 56 PM" src="https://user-images.githubusercontent.com/21316537/159203576-39ee2bba-41a8-4634-b56e-a2007ba66ffc.png">
<img width="538" alt="Screen Shot 2022-03-20 at 6 21 35 PM" src="https://user-images.githubusercontent.com/21316537/159203577-fed6ecf2-2dee-46d5-9317-230c702cb244.png">